### PR TITLE
no more lies - vamp rejuvenate function now matches it's description

### LINKED ISSUE #11111: Vampire rejuvenate doesnt flush out chloral
(WEB REPORT BY: vikenz REMOTE: 172.93.109.202:7777)
Revision: c0c03a45f16b91f45580fdec4051575fa1911156

Rejuvenate didnt work on a bunch of shit the cmo injected me with

> What you expected to happen
> To assblast the CMO
> 
> What actually happened
> A billion units of chloral and some other shit stayed in my body

Trialmin JustSumGuy replicated it and rejuvenate didnt work for him either

### PR DESCRIPTION
**Pull request by coldcola**
vampire rejuvenate skill will now flush all reagents from the user.
I did test this.

fixes #11111

### CODE REVIEW

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -99,6 +99,7 @@
 		M.current.weakened = 0
 		M.current.stunned = 0
 		M.current.paralysis = 0
+		M.current.reagents.clear_reagents()
```
Comment (ghost) [inline code/game/gamemodes/vampire/vampire_powers.dm:102]: Wait, holy fuck. This would clear holy water from them as well.

Comment (gbasood) [reply code/game/gamemodes/vampire/vampire_powers.dm:102]: !!!!

Comment (coldcola) [reply code/game/gamemodes/vampire/vampire_powers.dm:102]: to_chat(M.current, "<span class='notice'>You flush your system with clean blood and remove any incapacitating effects.</span>")
I have asked many times for a consensus on this to no response, should the skill match it's odd and overpowered description or should the description be changed?  I could increase the cooldown ? 
As it stands it fits the description, and is a buff.

Comment (Shadowmech88) [reply code/game/gamemodes/vampire/vampire_powers.dm:102]: Maybe just exempt holy water from the effect?

Comment (coldcola) [reply code/game/gamemodes/vampire/vampire_powers.dm:102]: I could do that, or I could increase the cooldown on the skill, most vampires I see killed end up burning in the chapel.

**Pull request merged**